src/Controls/tests/TestCases.HostApp/Issues/Issue24547.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 24547, "[Windows] FlyoutPage ShouldShowToolbarButton when overridden to return false, still shows button in title bar", PlatformAffected.UWP)]
+	public class Issue24547 : NavigationPage
+	{
+	
+		public Issue24547() : base(new Issue24547PopoverPage())
+		{
+
+		}
+		public partial class Issue24547PopoverPage : FlyoutPage
```

Comment:
```
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue24547.cs(4,15): error CS0260: Missing partial modifier on declaration of type 'Issue24547'; another partial declaration of this type exists [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue24547.cs(4,15): error CS0260: Missing partial modifier on declaration of type 'Issue24547'; another partial declaration of this type exists [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
    8570 Warning(s)
    2 Error(s)

```

Reply:
I have removed the partial keyword, could you please check once and let me know.